src/server/macos.rs

```diff
@@ -334,17 +334,28 @@ impl<'os> Method for PackageMethod<'os, Macos> {
         unix::upgrade(todo, options, self)
     }
     fn destroy(&self, options: &Destroy) -> anyhow::Result<()> {
-        let mut found = false;
-        log::info!(target: "edgedb::server::destroy",
-            "Unloading service");
-        let unit_path = unit_path(&options.name)?;
-        process::run(&mut StdCommand::new("launchctl")
-            .arg("unload").arg(&unit_path))?;
-        if unit_path.exists() {
+        let mut found;
+        let mut unit_file = unit_path(&options.name, true)?;
+        if unit_file.exists() {
```

Comment:
Here we run `unload` regardless of the existing file, because it possible that file was deleted by someone, but service is still running in memory. So I prefer to run `unload` (or bootout, or whatever) command anyway, to make sure we cleaned out all leftovers.

Reply:
You're right - looks like `launchctl bootout` supports taking a service target, I'll use that directly.